IBAnimatable/FlipAnimator.swift

```diff
@@ -21,14 +21,13 @@ public class FlipAnimator: NSObject, AnimatedTransitioning {
   fileprivate var horizontal: Bool = false
 
   // MARK: - Life cycle
-  public init(from direction: TransitionAnimationType.Direction, transitionDuration: Duration) {
+  public init(from direction: TransitionAnimationType.Direction, duration: Duration) {
```

Comment:
Ok I did this plus changed the remaining 3 that were not changed as to keep the API consistent. Thank you for clarifying @tbaranes